test/fixed.jl

```diff
@@ -69,7 +69,16 @@ struct_eq(A::AbstractSparseVector, B::AbstractSparseVector) =
     @test typeof(B) == typeof(F)
     @test struct_eq(B, F)
 end
+@testset "SparseMatrixCSC conversions" begin
+    A = sprandn(10, 10, 0.3)
+    F = fixed(copy(A))
+    B = SparseMatrixCSC(F)
+    @assert A == B
```

Comment:
Shouldn't this be

```suggestion
    @test A == B
```
? At least, I've never seen people testing with `@assert`, and I'm not sure this gets counted in code coverage.

Reply:
:laughing: i'm sorry, i think it gets counted but yeah, i shouldn't